src/endpoints/http/collections/list.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Get the collection list and their info (including the namespace, name, count and size). Returns a promise.
+ * @param {string} appname The appname. It should be in the format of "<domain>-<appGuid>-<envId>".
+ * @param {object} logger The logger object
+ * @param {db} the db connection
+ * @returns Promise
+ */
+export default function listCollections(appname, logger, db) {
+  logger.debug({appname: appname}, "list collections for app");
+  return db.listCollections().toArray().then(items => {
+    const promises = items.filter(item => item.name.indexOf('system.') === -1).map(item => db.collection(item.name).stats());
```

Comment:
that's true, we can show them to users. Not sure if there will be any side effects, but we can try it.